import-export-cli/integration/testutils/operationPolicy_testUtils.go

```diff
@@ -0,0 +1,552 @@
+	t.Helper()
+
+	pathToPolicyDirectory, _ := filepath.Abs(DevFirstSampleCaseMalformedOperationPolicyArtifactPath)
+	pathToPolicyhSpecFile, _ := filepath.Abs(DevSampleCaseMalformedOperationPolicyDefinitionPath)
+
+	args.ImportFilePath = pathToPolicyDirectory + "/"
+
+	assert.True(t, base.IsFileAvailable(t, pathToPolicyhSpecFile))
+
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Import API Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	importOutput, err := importAPIPolicy(t, args)
+
+	assert.NotNil(t, err, "Error while importing the API Policy")
+	assert.Contains(t, importOutput, "500", "Error importing API Policy")
+
+	changeExportedAPIPolicyFile(t, pathToPolicyhSpecFile)
+
+	importOutput, err = importAPIPolicy(t, args)
+
+	assert.NotNil(t, err, "Error while importing the API Policy")
+	assert.Contains(t, importOutput, "500", "Error importing API Policy")
+}
+
+// ValidateMalformedAPIPolicyExportImport : Validates Exporting API Policy from source env and Importing to destination env
+func ValidateAPIPolicyImportFailureWhenPolicyExisted(t *testing.T, args *PolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[PolicyNameKey])
+	policyVersion := fmt.Sprintf("%v", args.Policy[PolicyVersionKey])
+
+	args.SrcAPIM.Login(args.CtlUser.Username, args.CtlUser.Password)
+
+	exportedOutput, _ := exportAPIPolicy(t, policyName, args.SrcAPIM.GetEnvName())
+
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+
+	fmt.Println("Exported Path: ", args.ImportFilePath)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+
+	args.SrcAPIM.DeleteAPIPolicy(fmt.Sprintf("%v", args.Policy[PolicyIdKey]), "Export")
+
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Import API Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	args.DestAPIM.Login(args.CtlUser.Username, args.CtlUser.Password)
+
+	importedOutput, err := importAPIPolicy(t, args)
+
+	policyId := args.DestAPIM.GetAPIPolicyID(t, policyName, policyVersion)
+
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully Imported API Policy.")
+
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	importedOutput, _ = importAPIPolicy(t, args)
+
+	// assert.NotNil(t, err, "Error importing API Policy")
+	assert.Contains(t, importedOutput, "Error importing API Policy")
+
+	cleanUpImportExportPolicies(t, args, policyId)
```

Comment:
We shouldn't use explicit functions like this. Need to add t.Cleanups inside exportAPIPolicy() and importAPIPolicy() functions.